src/Controls/src/Core/ViewExtensions.cs

```diff
@@ -170,8 +171,13 @@ public static Task<bool> TranslateTo(this VisualElement view, double x, double y
 
 		internal static IAnimationManager? GetAnimationManager(this IAnimatable animatable)
 		{
-			if (animatable is Element e && e.FindParentOfType<Window>(true) is Window window)
-				return window.AnimationManager;
+			if (animatable is Element e && e.FindMauiContext() is IMauiContext mauiContext)
+			{
+				if (mauiContext is IScopedMauiContext scoped)
+					return scoped.AnimationManager;
+				else
+					return mauiContext.Services.GetService<IAnimationManager>();
+			}
```

Comment:
This has changed. Right now we first check for `IScopedMauiContext`, and if it is not that (for some reason) fall back to the services. Once we get scoped services, the internal interface will go away and we can keep the else branch.